src/components/common/CertifySection.vue

```diff
@@ -3,13 +3,14 @@
     <header>
       <h2>2. Certify</h2>
     </header>
-    <span>Enter the legal name of the person authorized to complete and submit these changes.</span>
+    <div class="pt-4">Enter the legal name of the person authorized to complete and submit these changes.</div>
     <certify
       :currentDate="getCurrentDate"
       :certified-by="getCertifyState.certifiedBy"
       :is-certified="getCertifyState.valid"
       :entity-display="readableEntityType"
       :message="certifyMessage"
+      :is-staff="isRoleStaff"
```

Comment:
Please use camelCase for props (and events and other properties).

Reply:
Done

Reply:
Whoops, now I noticed the other 3 props in kebab-case that should be camelCase.